odl/tomo/operators/ray_trafo.py

```diff
@@ -146,6 +149,21 @@ def __init__(self, reco_space, geometry, variant, **kwargs):
             if geometry.ndim > 2 and impl.endswith('cpu'):
                 raise ValueError('`impl` {!r}, only works for 2d'
                                  ''.format(impl_in))
+
+            # Filter axis-alinged initial detector position in parallel 3d
+            # geometry since it's broken for some ASTRA versions
+            if isinstance(geometry, Parallel3dAxisGeometry):
+                axis = geometry.axis
+                init_vec = (geometry._det_init_pos /
+                            np.linalg.norm(geometry._det_init_pos))
+                # Almost the same
+                if (np.isclose(np.dot(axis, init_vec), 1) and
+                        not astra_supports('par3d_axis_aligned_det_init_pos')):
```

Comment:
This check is not correct. I just installed ASTRA 1.7.1 which shows the issue with the default setup in [the 3d parallel beam example](https://github.com/odlgroup/odl/blob/master/examples/tomo/ray_trafo_parallel_3d.py). Here it is `axis = [0, 0, 1]` and `det_init_pos = `[1, 0, 0]`.
So `axis_aligned_det_init_pos` (taken from the comment in the example) is the wrong name. What is the exact fault scenario? Do you remember, @adler-j? The ASTRA issue is [here](https://github.com/astra-toolbox/astra-toolbox/issues/18) for reference.